README.md

```diff
@@ -1,15 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# LeaderBoard
 
 > Description the project.
+- In this project can store the name and the score of a person and display as a list.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JavaScript
 
 ## Live Demo (if available)
```

Comment:
- [ ] Live demo link is provided but leads to a broken page, if life link is nor ready yet, you can write something like, `coming soon..` rather than providing a broken link.